core/src/main/java/org/frankframework/configuration/digester/ValidateAttributeRule.java

```diff
@@ -161,10 +173,25 @@ private void checkDeprecationAndConfigurationWarning(String name, Method m) {
 			}
 
 			if (isDeprecated) {
-				addSuppressableWarning(msg, SuppressKeys.DEPRECATION_SUPPRESS_KEY);
+				addSuppressibleWarning(msg, SuppressKeys.DEPRECATION_SUPPRESS_KEY);
 			} else {
 				addLocalWarning(msg);
 			}
+			return;
+		}
+		// Check enum Configuration Warnings
+		if (!m.getParameters()[0].getType().isEnum())
+			return; // Skip non-enum setters
+		try {
+			Object o = ClassUtils.parseValueToSet(m, value);
+			if (o instanceof Enum<?> enumValue) {
+				String configWarning = getEnumConfigurationWarning(enumValue);
+				if (configWarning != null) {
+					addSuppressibleWarning("attribute [" + name + "." + enumValue + "]: " + configWarning, SuppressKeys.CONFIGURATION_VALIDATION);
+				}
+			}
+		} catch (IllegalArgumentException ignored) {
```

Comment:
ignored. Ik stel voor om even in een comment toe te voegen waarom dan

Reply:
Done.